curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-fetch-promises-and-async-await-by-building-an-fcc-forum-leaderboard/643c8ee68f280c0a58a3d439.md

```diff
@@ -0,0 +1,153 @@
+---
+id: 643c8ee68f280c0a58a3d439
+title: Step 29
+challengeType: 0
+dashedName: step-29
+---
+
+# --description--
+
+To print the result on the page, target the HTML element with the `id` of `posts-container`.
```

Comment:
```suggestion
Use the `.getElementById()` method to get the `#posts-container` element. Assign it to a `postsContainer` variable.
```

This step doesn't actually print the result on the page. (:

Reply:
updated